ext/gd/config.w32

```diff
@@ -56,16 +56,16 @@ if (PHP_GD != "no") {
 			gd_filter.c gd_pixelate.c gd_rotate.c gd_color_match.c gd_webp.c gd_avif.c \
 			gd_crop.c gd_interpolation.c gd_matrix.c gd_bmp.c gd_tga.c", "gd");
 		AC_DEFINE('HAVE_GD_BUNDLED', 1, "Define to 1 if gd extension uses GD library bundled in PHP.");
-		AC_DEFINE('HAVE_GD_PNG', 1, "PNG support");
-		AC_DEFINE('HAVE_GD_BMP', 1, "BMP support");
-		AC_DEFINE('HAVE_GD_TGA', 1, "TGA support");
-		AC_DEFINE('HAVE_LIBPNG', 1, "PNG support");
-		AC_DEFINE('HAVE_LIBJPEG', 1, "JPEG support");
-		AC_DEFINE('HAVE_GD_JPG', 1, "JPEG support");
-		AC_DEFINE('HAVE_XPM', 1, "XPM support");
-		AC_DEFINE('HAVE_GD_XPM', 1, "XPM support");
-		AC_DEFINE('HAVE_LIBFREETYPE', 1, "Freetype support");
-		AC_DEFINE('HAVE_GD_FREETYPE', 1, "Freetype support");
+		AC_DEFINE('HAVE_GD_PNG', 1, "Define to 1 if gd extension has PNG support.");
+		AC_DEFINE('HAVE_GD_BMP', 1, "Define to 1 if gd extension has BMP support.");
+		AC_DEFINE('HAVE_GD_TGA', 1, "Define to 1 if gd extension has TGA support.");
+		AC_DEFINE('HAVE_LIBPNG', 1, "Define to 1 if you have the libpng library.");
+		AC_DEFINE('HAVE_LIBJPEG', 1, "Define to 1 if you have the libjpeg library.");
+		AC_DEFINE('HAVE_GD_JPG', 1, "Define to 1 if gd extension has JPEG support.");
+		AC_DEFINE('HAVE_XPM', 1, "Define to 1 if you have the xpm library.");
+		AC_DEFINE('HAVE_GD_XPM', 1, "Define to 1 if gd extension has XPM support.");
+		AC_DEFINE('HAVE_LIBFREETYPE', 1, "Define to 1 if you have the FreeType library.");
+		AC_DEFINE('HAVE_GD_FREETYPE', 1, "Define to 1 if gd extension has FreeType support.");
```

Comment:
Oh, I thought that the bundled libgd also has JIT-mapped Japanese font support on Windows, but apparently not. For better compatibility with system libgd, we may consider to add that; on the other hand, this is a feature which always looked somewhat arbitrary to me – why not have that for other character encodings, too. Or maybe better, why not leave converting to UTF-8 up to users?

Anyhow, again unrelated to the PR at hand.